### PR TITLE
Added categories dropdownlist, changed some styling and added a Cours…

### PR DESCRIPTION
**Pull request by Omar-Alkhatib**
…e component

### CODE REVIEW

File: Front-End/src/components/Course copy.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+				<h3>Android Devlopment</h3>
+				<h3>Android Devlopment</h3>
+				<h3>Android Devlopment</h3>
+				<h3>Android Devlopment</h3>
+			</div>
+		</div>
+	);
+};
+
+export default Course;
```
Comment (Besslan) [inline Front-End/src/components/Course copy.js]: Why did you copy the course file?

File: Front-End/src/components/Course.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import logo from './pics/logo.png';
+
+const Course = (props) => {
+	console.log(props);
```
Comment (Besslan) [inline Front-End/src/components/Course.js]: Remove the console log

File: Front-End/src/components/Course.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+			</div>
+			<div className='course-details'>
+				<h2>Android Devlopment</h2>
+				<h3>Android Devlopment Android Devlopment Android Devlopment</h3>
+				<h3>Android Devlopment Android Devlopment Android Devlopment</h3>
+				<h3>Android Devlopment Android Devlopment Android Devlopment</h3>
+				<h3>Android Devlopment</h3>
+				<h3>Android Devlopment</h3>
+				<h3>Android Devlopment</h3>
+				<h3>Android Devlopment</h3>
```
Comment (Besslan) [inline Front-End/src/components/Course.js:18]: Why are these hard coded?

Comment (Omar-Alkhatib) [reply Front-End/src/components/Course.js:18]: Just as a place holder to create the general layout of the component. 
Will be dynamic in the next PR.

**Comment by Omar-Alkhatib** [comment]
All above required corrections have just been pushed to branch. 
Thanks.

**Pull request merged**